internal/usage/consts.go

```diff
@@ -15,4 +15,6 @@ const (
 	VipDhcpAllocationUsage string = "VIP auto alloc."
 	//usage of disk selection
 	DiskSelectionUsage string = "Disk Selection"
+	//user networkType selection
+	NetworkTypeSelectionUsage = "NetworkType Selection"
```

Comment:
you missed the string type and please change it to: Network Type Selection - this is the string that will appear in the dashboard

Reply:
Should be NetworkType only. These are labels so the shortest description is preferable